### ISSUE TITLE
Devtools integration

### ISSUE THREAD

**Issue by huang-julien** [nuxt-modules/i18n#2088]
### Describe the feature

Would be nice to have a `@nuxt/devtools` integration for `nuxt-i18n`

### Additional information

- [x] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by huang-julien** [comment]
What could be nice to have:
- all messages with all translations key by language (column)
````md
| key        | fr                 | en          | cn         |
|------------|--------------------|-------------|------------|
| HELLO_WORD | Bonjour le monde ! | Hello world | 你好世界   |
| HELLO      | Bonjour            | Hello       | 你好       |
| LOVE_NUXT  | J'adore nuxt       | I love nuxt | 我喜欢Nuxt |
````
- filter translations with an input
- See translations available in a component
- An on/off switch so `t` would only return the key in dev -> `i18n.t('HELLO_WORD ') === 'HELLO_WORD '
- edit message object (and save it)

**Comment by kazupon** [comment]
@huang-julien 
I have previously created a similar Vue CLI plugin that can manage resources.
Here's a screenshot of it.
Oh! I miss it!
![image.png](https://volta.s3.fr-par.scw.cloud/image_2602cc616e.png)

My interest as a devtools for i18n is interested in making it seamless for developers and translators.
Here is something I have prototyped before. (probably doesn't work now 😅 ).
https://github.com/intlify/devtools

The screenshot is here
![image.png](https://volta.s3.fr-par.scw.cloud/image_85b5b92d23.png)

In its devtools, it actually ties the text of the page to the key of the resource.
Recent localization services are able to link resource keys to screenshots.
I tried to improve the localize experience and developer experience through this prototype.
When I made this prototype, it was before GPT (Generative AI) came out.

I'm looking to find if I can create new devtools. Maybe Nuxt Devtools will inspire for me :)

**Comment by huang-julien** [comment]
Yes that's exactly what I was thinking about ! Had a lot of good memories with the cli plugin for i18n !
